ci/testnet-manager.sh

```diff
@@ -212,71 +214,80 @@ start() {
       #  and the leader might be in that subset)
       NO_VALIDATOR_SANITY=1 \
       RUST_LOG=solana=info \
-        ci/testnet-deploy.sh beta-testnet-solana-com ec2 us-west-1a \
-          -t "$CHANNEL_OR_TAG" -n 35 -c 0 -s -u -P -a eipalloc-0f286cf8a0771ce35 -D
+        ci/testnet-deploy.sh \
+          -t "$CHANNEL_OR_TAG" -n 50 -c 0 -s -u -P -a eipalloc-0f286cf8a0771ce35 -D \
+          beta-testnet-solana-com ec2 sa-east-1a us-west-1a ap-northeast-2a eu-central-1a ca-central-1a
       NO_VALIDATOR_SANITY=1 \
       RUST_LOG=solana=info \
-        ci/testnet-deploy.sh beta-testnet-solana-com gce us-west1-a \
-          -t "$CHANNEL_OR_TAG" -n 65 -c 0 -x -P -D
+        ci/testnet-deploy.sh \
+          -t "$CHANNEL_OR_TAG" -n 50 -c 0 -x -P -D \
+          beta-testnet-solana-com gce us-west1-b asia-east2-a europe-west4-a southamerica-east1-b us-east4-c
       NO_VALIDATOR_SANITY=1 \
       RUST_LOG=solana=info \
-        ci/testnet-deploy.sh beta-testnet-solana-com ec2 us-west-1a \
-          -t "$CHANNEL_OR_TAG" -n 35 -c 0 -s -u -P -a eipalloc-0f286cf8a0771ce35 \
+        ci/testnet-deploy.sh \
+          -t "$CHANNEL_OR_TAG" -n 50 -c 0 -s -u -P -a eipalloc-0f286cf8a0771ce35 \
           ${maybeReuseLedger:+-r} \
-          ${maybeDelete:+-D}
+          ${maybeDelete:+-D} \
+          beta-testnet-solana-com ec2 sa-east-1a us-west-1a ap-northeast-2a eu-central-1a ca-central-1a
       NO_VALIDATOR_SANITY=1 \
       RUST_LOG=solana=info \
-        ci/testnet-deploy.sh beta-testnet-solana-com gce us-west1-a \
-          -t "$CHANNEL_OR_TAG" -n 65 -c 0 -x -P \
+        ci/testnet-deploy.sh \
+          -t "$CHANNEL_OR_TAG" -n 50 -c 0 -x -P \
```

Comment:
I am hitting quota limits on GCP with 50 nodes. Let's reduce this to 40, and increase EC2 to 60.